ui/src/pages/Logs/Logs.tsx

```diff
@@ -217,14 +221,20 @@ export default function Logs() {
               </TableRow>
             </TableHead>
             <TableBody>
-              {filteredLogs.map((logInfo) => (
-                <LogsRow
-                  logInfo={logInfo}
-                  containerLabelColor={containerLabelColor}
-                  containerIconColor={containerIconColor}
-                  ddClient={ddClient}
-                />
-              ))}
+              {isLoading ? (
+                <Box sx={{ display: 'flex', justifyContent: 'center', width: '100vw' }}>
```

Comment:
A Box component should by default fill the horizontal space of its parent. So we should not override this with `width: "100vw"` which sets the width to the viewport width.

Instead, can you use these attributes, so that it will exactly match the Stats component?
```jsx
<Box sx={{ p: 4, display: 'flex', justifyContent: 'center' }}>
  <CircularProgress />
</Box>
```

Reply:
Can the conditional rendering be for the **entire table** instead of just the Rows? 

This will also keep consistent with the approach in the Stats component